### PR TITLE
Add doc examples for using UMI and SMI for SQL

### PR DESCRIPTION
**Pull request by seantleonard**
add examples for how to configure connection string for User assigned managed identity and system assigned managed identity.

### CODE REVIEW

**Comment by Jak-MS** [comment]
@seesharprun 
Can you review the proposed changes? 

**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by seesharprun — APPROVED** [review]

**Comment by seesharprun** [comment]
#sign-off

**Comment by seesharprun** [comment]
@Jak-MS, this is one of the owners of the documentation and shouldn't need approval from me typically.

File: data-api-builder/reference-configuration.md [hunk 1]
```diff
@@ -268,7 +268,12 @@ Using this formula, you can calculate the time for each retry attempt in seconds
...
+
+- **User Assigned Managed Identity** (UMI): Append the *Authentication* and *User Id* properties to your connection string while substituting in your User Assigned Managed Identity's client id: `Authentication=Active Directory Managed Identity; User Id=<UMI_CLIENT_ID>;`.
+- **System Assigned Managed Identity** (SMI): Append the *Authentication* property and exclude the *UserId* and *Password* arguments from your connection string: `Authentication=Active Directory Managed Identity;`. The absence of the *UserId* and *Password* connection string properties will signal DAB to authenticate using a system assigned managed identity.
+
+For more information about configuring a Managed Service Identity with Azure SQL or SQL Server, see [Managed identities in Microsoft Entra for Azure SQL](/azure/azure-sql/database/authentication-azure-ad-user-assigned-managed-identity?view=azuresql).
```
Comment (Jak-MS) [inline data-api-builder/reference-configuration.md]:
```suggestion
For more information about configuring a Managed Service Identity with Azure SQL or SQL Server, see [Managed identities in Microsoft Entra for Azure SQL](/azure/azure-sql/database/authentication-azure-ad-user-assigned-managed-identity?view=azuresql&preserve-view=true).
```

**Comment by Jak-MS** [comment]
@seesharprun - understood and thanks ... Note, however, that the rules for merging in the public repo are fairly strict and require approval / sign off of the article owner ... @seantleonard can send a note to @MicrosoftDocs/public-repo-pr-review-team each time or use the private repo (where no approval is needed by you).

**Comment by Jak-MS** [comment]
fyi @seantleonard - in case you'd like to use the private repo, here's some info:

Read about [authors working in the public repo](https://review.docs.microsoft.com/help/contribute/contribute-authors-in-public-repo?branch=main) for more information. 

If you need help getting set up to work in the private repo, see [Get started contributing to the Microsoft Learn platform](https://review.learn.microsoft.com/help/get-started/?branch=main&tabs=learn).

**Comment by seantleonard** [comment]
@seesharprun && @Jak-MS , when do these changes publish ?

**Comment by Jak-MS** [comment]
@seantleonard - it looks like the changes were accidentally reverted... i opened a new PR https://github.com/MicrosoftDocs/data-api-builder-docs/pull/20

**Pull request merged**